### PR TITLE
Wordpress upgrades

### PR DESCRIPTION
**Pull request by ckraybill**
There is a legacy engineering site at http://engineering.g5platform.com. I have done some upgrades to the g5engsite source to support importing these articles.

Once deployed to production I can import the XML export, clean up the blog articles, and re-direct the legacy site to the new http://engineering.getg5.com site.

### CODE REVIEW

**Comment by variousred** [comment]
:shipit:

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: bundle exec rails server thin -p $PORT -e $RACK_ENV
```
Comment (libryder) [inline Procfile:1]: +1

File: README.markdown [hunk 2]
```diff
@@ -1,28 +1,101 @@
...
-1. Create a branch
-2. Commit changes to branch
-3. Submit a pull request
+1. Clone it
+2. Get it running
+3. Create your feature branch (`git checkout -b my-new-feature`)
+4. Write your code and **specs**
+5. Commit your changes (`git commit -am 'Add some feature'`)
+6. Push to the branch (`git push origin my-new-feature`)
+7. Create new Pull Request
```
Comment (libryder) [inline README.markdown:101]: Thank you 1000x for updating the README!

Comment (ckraybill) [inline README.markdown:101]: You got top billing as the largest contributor :smile:

**Comment by libryder** [comment]
:shipit:

**Pull request merged**